aea/aea.py

```diff
@@ -74,6 +74,7 @@ def __init__(
         period: float = 0.05,
         execution_timeout: float = 0,
         max_reactions: int = 20,
+        error_handler_class: Optional[Type] = None,
```

Comment:
ErrorHandler needs to have an ABC in this case. Or how do we enforce the interface?

Reply:
Yes